README.md

```diff
@@ -458,7 +458,7 @@ cdnvm(){
         nvm_version=$(<"$nvm_path"/.nvmrc)
 
         # Add the `v` suffix if it does not exists in the .nvmrc file
-        if [[ $nvm_version != v* ]]; then
+        if [[ $nvm_version != v* && $nvm_version != lts/* && $nvm_version != iojs* && $nvm_version != node && $nvm_version != system && $nvm_version != stable && $nvm_version != default ]]; then
             nvm_version="v""$nvm_version"
         fi
 
```

Comment:
below this, `nvm use` isn't needed if `nvm install` has ran; the two `if`s should instead be one `if..elif`.

Reply:
Updated with commit 20bbb090746a71544be74da9c18ef7e20c05b184